src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -10,8 +10,11 @@
     <TreatSpecificWarningsAsErrors />
   </PropertyGroup>
   <ItemGroup>
+    <Compile Remove="NHSpecificTest\GH01226\**" />
+    <EmbeddedResource Remove="NHSpecificTest\GH01226\**" />
```

Comment:
Hi,
I do not think the changes on the test csproj are needed. Please revert them or explain them.

Reply:
Yes, it is wrong lines. I will fix them